custom_components/tuya_local/devices/pgst_pw150_alarm.yaml

```diff
@@ -0,0 +1,195 @@
+        - dps_val: arm
+          value: false
+        - dps_val: home
+          value: false
+        - dps_val: sos
+          value: true
+        - value_redirect: alarm_state
+    - id: 24
+      type: string
+      name: zone_attribute
+      optional: true
+    - id: 32
+      type: string
+      name: master_state
+      optional: true
+    - id: 36
+      type: string
+      name: sub_class
+      optional: true
+    - id: 37
+      type: string
+      name: sub_type
+      optional: true
+    - id: 38
+      type: string
+      name: sub_admin
+      optional: true
+    - id: 39
+      type: string
+      name: sub_state
+      optional: true
+      readonly: true
+    - id: 103
+      type: string
+      name: zone_number
+      optional: true
+      readonly: true
+    - id: 104
+      type: string
+      name: other_event
+      optional: true
+      readonly: true
+secondary_entities:
+  - entity: number
+    name: Exit delay
+    category: config
+    dps:
+      - id: 2
+        type: integer
+        name: value
+        unit: s
+        range:
+          min: 0
+          max: 600
+  - entity: number
+    name: Siren duration
+    category: config
+    dps:
+      - id: 3
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 600
+  - entity: switch
+    name: Alarm sound
+    category: config
+    dps:
+      - id: 4
+        type: boolean
+        name: switch
+  - entity: number
+    name: Voice volume
+    category: config
+    dps:
+      - id: 5
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 14
+  - entity: switch
+    name: Doorbell Chimes Forbid
+    category: config
+    dps:
+      - id: 12
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Silence siren
+    category: config
+    dps:
+      - id: 25
+        type: boolean
+        name: switch
+        optional: true
+  - entity: event
+    class: doorbell
+    category: diagnostic
+    dps:
+      - id: 26
+        type: string
+        name: event
+        optional: true
+        mapping:
+          - dps_val: "AEQAbwBvAHIAYgBlAGwAbAAgAFQAcgBpAGcAZwBlAHL/\
+              AQAKAFoAbwBuAGUAOgAwADA="
+            value: ring
+            # means: Doorbell Trigger! \n Zone:00
+          - value: null
+  - entity: number
+    name: Entry delay
+    category: config
+    dps:
+      - id: 28
+        type: integer
+        name: value
+        unit: s
+        range:
+          min: 0
+          max: 600
+  - entity: number
+    name: Doorbell volume
+    category: config
+    dps:
+      - id: 30
+        type: integer
+        name: value
+        range:
+          min: 0
+          max: 15
+  - entity: select
+    name: Language
+    icon: "mdi:translate"
+    category: config
+    dps:
+      - id: 40
+        type: string
+        name: option
+        mapping:
+          - dps_val: chinese_simplified
+            value: 中文简化字
```

Comment:
If you think this is a better translation than simply 中文, then it may be better to change the translation files.

Reply:
I copied that from somewhere and didn't question it. You're right and I've changed it generally regarding your comment below (use translation_key).

Reply:
It is a more literal translation, but since we already had a translation for "chinese", I reused that for translations where the tuya value is chinese_simplified, as in many cases it is the only Chinese option, and even when it isn't, only one of the two needs to be fully identified to disambiguate them (though there may be a political objection to not treating them equally, which is why I am not opposed to changing it).